pyuvdata/uvflag/uvflag.py

```diff
@@ -1802,6 +2252,9 @@ def __add__(
                 "jones": [2, 3, 4],
             }
         ax = axis_nums[axis][type_nums[self.type]]
+
+        # TODO: need to add a bunch of compatibility checking!
```

Comment:
is this still true?

Reply:
Yes, I think it is. That checking has been absent, I haven't added it as part of this PR but maybe I should.

Reply:
ok, I added that checking now and removed the TODO.